scripts/pre_commit_linter.py

```diff
@@ -470,15 +470,17 @@ def _redirect_stdout(new_target):
         sys.stdout = old_target
 
 
-def _get_all_files(input_files, input_path):
-    """This function is used to return the files which needs to linted and
+def _get_all_files(input_path, input_files):
+    """This function is used to return the files which needs to be linted and
     checked.
 
     Args:
-        input_files: list. The list of files to be linted and checked.
         input_path: str. The path of the directory to be linted and checked.
+        input_files: list(str). The list of files to be linted and checked,
+        ignored if input_path is specified.
```

Comment:
Indent continuation line by 4.

/cc @oppia/dev-workflow-team This should be a lint check.

Reply:
Filing an issue for the same.

Reply:
https://github.com/oppia/oppia/issues/6422